src/data/finset/n_ary.lean

```diff
@@ -213,6 +213,20 @@ lemma image_image₂_right_comm {f : α → β' → γ} {g : β → β'} {f' : 
   image₂ f s (t.image g) = (image₂ f' s t).image g' :=
 (image_image₂_distrib_right $ λ a b, (h_right_comm a b).symm).symm
 
+/-- The other direction does not hold because of the `s`-`s` cross terms on the RHS. -/
+lemma image₂_distrib_subset_left {γ : Type*} {u : finset γ} {f : α → δ → ε} {g : β → γ → δ}
```

Comment:
What's wrong with the `γ` at the top of the file?

Reply:
It assumes `decidable_eq γ`.